src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Books from './Books';
+
+const BookContent = ({ books }) => (
+  <ul className="book-content">
+    {
+books.map((book) => <Books key={book.id} book={book} />)
+}
+  </ul>
```

Comment:
[OPTIONAL] I recommend changing the name of this component to `BooksList` to improve the readability of your code.